src/thunderbots/software/ai/intent/avoid_area.cpp

```diff
@@ -27,3 +27,54 @@ std::ostream& operator<<(std::ostream& os, const AvoidArea& area)
     }
     return os;
 }
+
+avoid_area_mask_t getAvoidAreasFromGameState(const GameState& game_state)
+{
+    avoid_area_mask_t areas_to_avoid = 0;
+
+    // default avoid areas regardless of game state
+    areas_to_avoid.set((uint32_t)AvoidArea::FRIENDLY_DEFENSE_AREA, true);
```

Comment:
see below, I think u can specify enum type and remove all the casting iirc

Reply:
not working, can you provide example?

Reply:
my bad, it doesn't work, enums don't allow implicit casting regardless of what type is specified